mlflow/server/js/src/MlflowRouter.tsx

```diff
@@ -34,22 +32,13 @@ const landingRoute = {
 /**
  * This is root element for MLflow routes, containing app header.
  */
-const MlflowRootRoute = ({
-  isDarkTheme,
-  setIsDarkTheme,
-  useChildRoutesOutlet = false,
-  routes,
-}: {
-  isDarkTheme?: boolean;
-  setIsDarkTheme?: (isDarkTheme: boolean) => void;
-  useChildRoutesOutlet?: boolean;
-  routes?: any[];
-}) => {
+const MlflowRootRoute = () => {
   useInitializeExperimentRunColors();
 
   const [showSidebar, setShowSidebar] = useState(true);
   const { theme } = useDesignSystemTheme();
   const { experimentId } = useParams();
+  const { isDarkTheme, setIsDarkTheme } = useDarkThemeContext();
```

Comment:
see [below comment](https://github.com/mlflow/mlflow/pull/19044/files#r2563694507)

```suggestion
  const { setIsDarkTheme } = useDarkThemeContext();
  const isDarkTheme = theme.isDarkTheme;
```